Orange/widgets/model/owneuralnetwork.py

```diff
@@ -154,20 +163,33 @@ def setup_layout(self):
         gui.button(self.apply_button, self, "Cancel", callback=self.cancel)
 
     def create_learner(self):
-        return self.LEARNER(
-            hidden_layer_sizes=self.get_hidden_layers(),
-            activation=self.activation[self.activation_index],
-            solver=self.solver[self.solver_index],
-            alpha=self.alpha,
-            max_iter=self.max_iterations,
-            preprocessors=self.preprocessors)
+        learner = None
```

Comment:
This is redundant; `learner` will be set to something in one of the following lines.

Reply:
Than we can drop it.